### PR TITLE
Expose opsets as part of PythonAPI

### PR DESCRIPTION
**Pull request by Ewa21**

### CODE REVIEW

File: ngraph/python/src/ngraph/ops/opset2/opset2.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
+# ******************************************************************************
```
Comment (mitruska) [inline ngraph/python/src/ngraph/ops/opset2/opset2.py:1]: `__init__.py` for opset2 is missing

File: ngraph/python/src/ngraph/ops/opset3/__init__.py [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+from ngraph.ops.opset1 import equal
+from ngraph.ops.opset1 import erf
+from ngraph.ops.opset1 import exp
+from ngraph.ops.opset1 import fake_quantize
+from ngraph.ops.opset1 import floor
+from ngraph.ops.opset1 import floor_mod
+from ngraph.ops.opset1 import gather
+from ngraph.ops.opset1 import gather_tree
+from ngraph.ops.opset2 import gelu
+from ngraph.ops.opset4 import get_output_element
```
Comment (mitruska) [inline ngraph/python/src/ngraph/ops/opset3/__init__.py:73]: Probably we shouldn't import op from higher opset, so `opset4` in `opset3`

File: ngraph/python/src/ngraph/ops/opset4/opset4.py [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    if opset_version:
+        return NodeFactory(opset_version)
+    else:
+        return NodeFactory()
+
+
+# ------------------------ ops ---------------------------------------------------
+
+@nameable_op
+def get_output_element(data: NodeInput, index: int, name: Optional[str] = None) -> Node:
```
Comment (mitruska) [inline ngraph/python/src/ngraph/ops/opset4/opset4.py:66]: In general we should think about removing `get_output_element` from the pythonAPI. 
Anyway, `opset4` is not the best place for this op.

File: ngraph/python/src/ngraph/ops/opset2/opset2.py (hunk 1 above)
Comment (Ewa21) [reply ngraph/python/src/ngraph/ops/opset2/opset2.py:1]: Done

Comment (Ewa21) [reply ngraph/python/src/ngraph/ops/opset2/opset2.py:1]: Done

File: ngraph/python/src/ngraph/ops/opset3/__init__.py (hunk 2 above)
Comment (Ewa21) [reply ngraph/python/src/ngraph/ops/opset3/__init__.py]: Done

File: ngraph/python/src/ngraph/ops/opset4/opset4.py (hunk 3 above)
Comment (Ewa21) [reply ngraph/python/src/ngraph/ops/opset4/opset4.py]: You're right. We can remove it.

File: ngraph/python/src/ngraph/opset1/ops.py [hunk 4]
```diff
@@ -3490,55 +2816,102 @@ def tensor_iterator(
...
     """
-    return _get_node_factory().create(
-        "ExtractImagePatches",
-        [as_node(image)],
-        {"sizes": sizes, "strides": strides, "rates": rates, "auto_pad": auto_pad},
-    )
+    return _get_node_factory_opset1().create("VariadicSplit", as_nodes(data, axis, split_lengths))
+
+
+# Extend Node class to support binary operators
```
Comment (postrational) [inline ngraph/python/src/ngraph/opset1/ops.py:2901]: Can we move this section to `ngraph/__init__.py`? This is executed at import-time and `opset1` could potentially not be imported. Circular imports could be an issue.

Comment (Ewa21) [reply ngraph/python/src/ngraph/opset1/ops.py]: Done

File: ngraph/python/src/ngraph/opset1/__init__.py [hunk 5]
```diff
@@ -0,0 +1,134 @@
...
+# ******************************************************************************
+"""ngraph module namespace, exposing factory functions for all ops and other classes."""
+# noqa: F401
+
+from pkg_resources import get_distribution, DistributionNotFound
+
+try:
+    __version__ = get_distribution("ngraph-core").version
+except DistributionNotFound:
+    __version__ = "0.0.0.dev0"
```
Comment (postrational) [inline ngraph/python/src/ngraph/opset1/__init__.py:24]: Please remove this code, it should only be in `ngraph/__init__.py`. It specifies the version of nGraph used.

Comment (Ewa21) [reply ngraph/python/src/ngraph/opset1/__init__.py]: Done

**Pull request merged**